src/main/java/org/openstreetmap/atlas/geography/converters/MultiplePolyLineToPolygonsConverterCommand.java

```diff
@@ -0,0 +1,62 @@
+package org.openstreetmap.atlas.geography.converters;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.openstreetmap.atlas.exception.CoreException;
+import org.openstreetmap.atlas.geography.PolyLine;
+import org.openstreetmap.atlas.geography.Polygon;
+import org.openstreetmap.atlas.streaming.resource.File;
+import org.openstreetmap.atlas.streaming.writers.SafeBufferedWriter;
+import org.openstreetmap.atlas.utilities.collections.Iterables;
+import org.openstreetmap.atlas.utilities.collections.StringList;
+import org.openstreetmap.atlas.utilities.runtime.Command;
+import org.openstreetmap.atlas.utilities.runtime.CommandMap;
+
+/**
+ * @author matthieun
+ */
+public class MultiplePolyLineToPolygonsConverterCommand extends Command
+{
+    private static final Switch<File> POLYLINES = new Switch<>("polylines",
+            "File containing lines of semicolon separated list of WKT polylines", File::new,
+            Optionality.REQUIRED);
+    private static final Switch<File> POLYGONS = new Switch<>("polygons",
+            "Output file that will contain lines of semicolon separated list of reconstructed polygons",
+            File::new, Optionality.REQUIRED);
```

Comment:
What about outputting to stdout if no output file is provided?

Reply:
Most commands of this type provide an output destination. Given large number of PolyLines, stdout will be overwhelmed. I'd keep as is.